access-control-app/docs/api.md

```diff
@@ -863,6 +864,19 @@ Content-Type: application/json;charset=ISO-8859-1
 {"C1200000000-PROV1": ["read"], "C1200000001-PROV1": []}
 ```
 
+When requesting permissions for large sets of concept ids, you can make a POST request to `%CMR-ENDPOINT%/permissions` and pass the query parameters in the POST body.
+
+Example POST request:
+
+```
+curl -i -XPOST -H "Content-Type: application/json" "%CMR-ENDPOINT%/permissions" -d 'user_type=guest&concept_id=C1200000003-PROV1&concept_id=C1200000004-PROV1'
```

Comment:
can we add some new lines ("\") to break this up and keep it from needing a scroll?

Reply:
Moved the query body to its own line.